core/src/main/java/org/elasticsearch/rest/action/cat/RestNodesAction.java

```diff
@@ -85,7 +85,7 @@ public RestChannelConsumer doCatRequest(final RestRequest request, final NodeCli
         clusterStateRequest.clear().nodes(true);
         clusterStateRequest.local(request.paramAsBoolean("local", clusterStateRequest.local()));
         clusterStateRequest.masterNodeTimeout(request.paramAsTime("master_timeout", clusterStateRequest.masterNodeTimeout()));
-
+        final boolean fullId = request.paramAsBoolean("full_id", false);
```

Comment:
yeah that's fine - it's an exception and that is why you likely added this. In this case we have no exception and doing it without hiding this parameter in some special method was possible. I strongly encourage people to do it this way. I don't think this falls under _taste_ we can't hide stuff just because we have an abstraction